src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -326,7 +326,7 @@ void GeneralPage::initLayout(GeneralPageView &layout)
 
     ComboBox *dankDropdown =
         layout.addDropdown<std::underlying_type<StreamerModeSetting>::type>(
-            "Enable Streamer Mode", {"No", "Yes", "Detect OBS (Windows only)"},
+            "Enable Streamer Mode", {"No", "Yes", "Detect OBS"},
```

Comment:
Think we should mention that is does not work on Mac (or maybe the Linux way could also be used on Mac?)